slave/meta/rpm.dependencies

```diff
@@ -2,6 +2,7 @@ perun-slave-base
 perun-slave-process-afs
 perun-slave-process-afs-group
 perun-slave-process-apache-basic-auth
+perun-slave-process-apache-shibboleth
```

Comment:
Adding here is ok, since this meta-package should install all perun services. It's a good point, because we recently forgot to add new services here. Thank you for that.